lib/active_model/type/salesforce/multipicklist.rb

```diff
@@ -3,20 +3,27 @@
 module ActiveModel
   module Type
     module Salesforce
-      class Mulitpicklist < ActiveModel::Type::Value
+      class Multipicklist < ActiveModel::Type::Value
+        include ActiveModel::Type::Helpers::Mutable
```

Comment:
This fixes a bug where if you mutate an existing array, it won't catch that it changed if you were to save the record.  You would've have to set new array in order for the change value to be picked up.